Dockerfile

```diff
@@ -26,3 +26,8 @@ COPY *.cfg *.conf *.sh *.in *.txt *.py ${PROJECT_HOME}/
 
 # tests
 COPY .pylintrc .flake8 ${PROJECT_HOME}/
+
+# labels
+ARG commit
+LABEL org.opencontainers.image.source="https://github.com/elifesciences/sciencebeam"
```

Comment:
What about the remote URL? Jenkins knows where to get it from. Jenkins might already know the answer so that it doesn't need to be hard-coded here?

Reply:
Jenkins wouldn't use this label, nor in general we have use cases (yet) where we retrieve the source code from a Docker image

Reply:
Apparently no effect in Docker Hub. Needs further reading / testing.